config/samples/gcp/operator_v1alpha1_externaldns_detailed.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: externaldns.olm.openshift.io/v1alpha1
+kind: ExternalDNS
+metadata:
+  name: sample-gcp
+spec:
+  # DNS provider
+  provider:
+    type: GCP
+    gcp:
+      credentials:
+        # secret should be in the operator namespace
+        name: gcp-access-key
+      project: openshift-gce-devel
```

Comment:
```suggestion
      # project auto detection doesn't work from within containers of some Kubernetes distros
      # specify the project explicitly to avoid this problem
      project: openshift-gce-devel
```

Reply:
done 👍🏽